### PR TITLE
fixing RemoteOverrideModule

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**
The module was requiring a variable, not a string

**What kind of change does this PR introduce?**

bugfix
**Did you add tests for your changes?**
no

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**
no

### CODE REVIEW

File: lib/container/RemoteOverrideModule.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ class RemoteModule extends Module {
 		const externalModuleId = chunkGraph.getModuleId(externalModule);
 		let i = 0;
 		const source = Template.asString([
-			`var external = __webpack_require__(${externalModuleId});`,
+			`var external = __webpack_require__("${externalModuleId}");`,
```
Comment (ScriptedAlchemy) [inline lib/container/RemoteOverrideModule.js]: this failed when requiring `container/website2`

Comment (ScriptedAlchemy) [reply lib/container/RemoteOverrideModule.js]: it would come through as a variable not a string

**Comment by ScriptedAlchemy** [comment]
image

**Comment by sokra** [comment]
merged in #10486

**Pull request closed without merging**